### DISCUSSION TITLE
support for macbook?

### DISCUSSION THREAD

**Discussion by Tb0ne4** [OhMyGuus/BetterCrewLink#22] [Q&A] [answered]
is it possible to install and use this software on macbook? and if not, I would appreciate if it could be considered.

**Comment by OhMyGuus** [comment, marked as answer]
Throught I disabled the discussion tab, but apperently not xD. 
There will be a mac version soon that requires one pc player (or a bot soon) 

If you play with friends only tho then you can just use https://web.bettercrewl.ink/, but everyone needs to be on the voice server: `https://bettercrewl.ink` and one person on PC needs to enable Mobile host.

**Reply by symsym-1629** [reply]
is this outdated or is it planned ?

**Reply by MatadorProBr** [reply]
> is this outdated or is it planned ?

Planned but probably it will take some time, in the meantime you can just use https://web.bettercrewl.ink/ (as said on the OhMyGuus message)
